tests/fast-integration/package-lock.json

```diff
@@ -1,4446 +1,8 @@
 {
   "name": "kyma-js",
   "version": "0.0.1-alpha.12",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Try to upgrade the version of your `node.js`. I think we don't want to downgrade the version here